tests/shared/src/test/scala/spire/algebra/PartialOrderSuite.scala

```diff
@@ -7,11 +7,11 @@ class PartialOrderSuite extends munit.FunSuite {
   import spire.implicits._
 
   test("Minimal and maximal elements of {{1, 2, 3}, {3}, {2}, {1}} by power set partial order") {
-    val sets = Seq(Set(1, 2, 3), Set(3), Set(2), Set(1), Set(1, 4))
-    assertEquals(sets.pmin.toSet, Set(Set(1), Set(2), Set(3)))
-    assertEquals(sets.pmax.toSet, Set(Set(1, 2, 3), Set(1, 4)))
+    val sets = Seq(Set[Int](1, 2, 3), Set[Int](3), Set[Int](2), Set[Int](1), Set[Int](1, 4))
+    assertEquals(seqOps(sets).pmin.toSet, Set(Set[Int](1), Set[Int](2), Set[Int](3)))
+    assertEquals(seqOps(sets).pmax.toSet, Set(Set[Int](1, 2, 3), Set[Int](1, 4)))
```

Comment:
What's going on with `seqOps`?

Reply:
Fixed reverting the syntax to use implicit conversions